src/main/java/util/Engine.java

```diff
@@ -6,73 +6,123 @@
 
 import static org.lwjgl.glfw.GLFW.glfwInit;
 
+/**
+ * The Engine class initializes GLFW and the game loop. <br\>
+ * Singleton.
+ */
 public final class Engine {
-
     /**
-     * The Engine class initializes GLFW and the game loop.
+     * The static {@code Engine} instance.
      */
-
     private static final Engine instance = new Engine();
-
+    
+    /**
+     * The exact time the engine started in milliseconds.
+     */
     private final long startMillis;
+    
+    /**
+     * No clue
+     */
     private Window window;
+    
+    /**
+     * Whether the {@code Engine} is running.
+     */
```

Comment:
![image](https://user-images.githubusercontent.com/32278830/125672138-17a2c244-df1d-423c-90ab-894e56c44023.png)

Reply:
this seems clear that it is about whether or not the engine is running